src/components/NavBar/NavBar.css

```diff
@@ -13,7 +13,7 @@
   width: 50px;
 }
 
-.navbar-buttons-and-account {
```

Comment:
Not sure if this was an accident, but please add back in the "navbar" at the beginning of all the classnames in this file. This is removing the styling from the navbar.